open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/mappers/relationships/IsARelationshipMapper.java

```diff
@@ -46,6 +46,7 @@ protected void mapRelationshipToInstanceProperties(IsA iSARelationship, Instance
         if (iSARelationship.getStatus() != null) {
             EnumPropertyValue enumPropertyValue = new EnumPropertyValue();
             enumPropertyValue.setOrdinal(iSARelationship.getStatus().getOrdinal());
+            enumPropertyValue.setSymbolicName(iSARelationship.getStatus().getName());
```

Comment:
I am not sure what difference this change I assume it is for the UI, so it displays the content more descriptively, if so then it should be on all of the mappers. It would make sense to have a method in SubjectAreaUtils similar to the other existing ones but for status:

it would be called like this:
   SubjectAreaUtils.setStatusPropertyInInstanceProperties(instanceProperties, iSARelationship.getStatus(), OpenMetadataAPIMapper.STATUS_PROPERTY_NAME);

Reply:
The fix was not for giving descriptive information. In the flow of creating the IsA relationship in Egeria Core, there is code that accesses the symbolicName of the 'status' Enum. This threw a nullpointer exception and stopped the creation of the relationship. Adding this code will prevent the exception.
I will refactor this code to the utils class and update my pull req.